### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Koliras**

### CODE REVIEW

**Review by kaavka — APPROVED** [review]
I dont even know what to add) Maybe just some identations between cases. Good job.

File: src/transformState.js [hunk 1]
```diff
@@ -5,7 +5,24 @@
...
+        break;
+      case 'removeProperties':
+        action.keysToRemove.forEach(key => {
+          delete state[key];
+        });
+        break;
+      case 'clear':
+        Object.keys(state).forEach(key => {
+          delete state[key];
+        });
```
Comment (qoa11a) [inline src/transformState.js:21]: It's one of the ways to delete all properties from an object. The other one is to assign an empty object to the `state` variable. In the second case, you'll create a new reference so they're not identical.

The first way is slower, creating a nested loop but keeping the same reference. The second way is faster, without a nested loop, but it creates a new reference.

File: src/transformState.js [hunk 2]
```diff
@@ -5,7 +5,24 @@
...
+        break;
+      default: break;
```
Comment (qoa11a) [inline src/transformState.js:23]:
```suggestion
      default: 
        break;
```
I'd recommend inserting a new line after any of the cases.

**Review by qoa11a — APPROVED** [review]
Nice!

**Pull request closed without merging**